packages/flutter/lib/src/widgets/view.dart

```diff
@@ -131,7 +131,7 @@ class View extends StatelessWidget {
   ///
   /// See also:
   ///
-  ///  * [View.maybeOf], which throws instead of returning null if no
+  ///  * [View.maybeOf], which returns null instead of throwing if no
```

Comment:
The implementation of this method sounds like it'll throw if result is null (either by the code of the assert, or by returning `result!`? Revert this change?

Reply:
This comment is referring to `View.maybeOf` which doesn't throw.

Reply:
Thanks for catching this!